themes/openy_themes/openy_carnation/openy_carnation.theme

```diff
@@ -651,3 +651,13 @@ function openy_carnation_entity_file_uri_by_fid($fid_array) {
 
   return '';
 }
+
+/**
+ * Implements hook_preprocess_video_embed_iframe().
+ */
+function openy_carnation_preprocess_video_embed_iframe(array &$variables) {
+  if ($attr = $variables['attributes']) {
+    /** @var \Drupal\Core\Template\Attribute $attr */
+    $attr->setAttribute('frameborder', 1);
```

Comment:
what if there is already frameborder, but not 1. It'll be overridden by this code.
Let's be less generic
If no frameborder - set it to 1
if frameborder = 0, or any other number - skip it

Reply:
@podarok sorry, but I don't understand that logic.
The frameborder attrubute by default has value 0, for all providers. And if we will skip it for frameborder = 0 , then our code doen't work at all
![image (18)](https://user-images.githubusercontent.com/744406/127912894-147c6a34-c553-4ec3-af57-b91d97b86e10.png)

In what case we can have another value for that attribute?
cc @hamrant

Reply:
@aleevas as a content manager I can set frameborder explicitly to 0. your code would break it and reset to 1
We need to check if frameborder is set to any value before setting it to 1 in our case.
If it is set to any value - we must avoid of setting to 1

Reply:
@podarok 
Sorry, but I can't find any ability for content manager to change value for farmeborder attribute.
As you can see on screenshot above, the have settings only for width and height attribute, the frameborder attribute has a hardcoded value = 0. And this is actual in any other placec where mentioning the framborder attribute even in others modules.
Why we expect the  border=1 as it written in jira issue?
Please help me to understand what I missed here
cc @hamrant

Reply:
No need to underestimate content managers
They can edit HTML in WYSIWYG in order to set 0 for frameborder. @aleevas

Reply:
@podarok 
thanks for you time.
I've added a new commit with logic that you presented in first comment of this thread.
Please check if I got you right
cc @hamrant